README.md

```diff
@@ -18,7 +18,7 @@ A mighty, modern CSS linter and fixer that helps you avoid errors and enforce co
 -   **Support for plugins:** It's easy to create your own rules and add them to the linter, or make use of plugins written by the community.
 -   **Automatically fixes some stylistic violations:** Save time by having stylelint fix your code with this *experimental* feature.
 -   **Shareable configs:** If you don't want to craft your own config, you can extend a shareable config. Or you can craft your own config and share with your team and/or the community.
--   **Supports *CSS-like* syntaxes:** The linter is powered by [PostCSS](https://github.com/postcss/postcss), so it can be configured and extended to understand any syntax that PostCSS can parse, including SCSS, [SugarSS](https://github.com/postcss/sugarss), and Less.
+-   **Parses *CSS-like* syntaxes:** The linter is powered by [PostCSS](https://github.com/postcss/postcss), so it can be configured and extended (e.g. [stylelint-scss](https://github.com/kristerkari/stylelint-scss)) to understand any syntax that PostCSS can parse, including SCSS, [SugarSS](https://github.com/postcss/sugarss), and Less.
```

Comment:
I'm not sure `stylelint-scss` should be there. It doesn't parse style sheet, it provides rules for non-standard syntaxes.

Reply:
Fair point